calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/data_collector.py

```diff
@@ -530,3 +585,62 @@ def process_detection(
             return CollectionStatus.ACCEPTED
 
         return CollectionStatus.REDUNDANT
+
+    def process_detection_eval_mode(
+        self,
+        image: np.array,
+        detection: BoardDetection,
+        camera_model: Optional[CameraModel] = None,
+        mode: OperationMode = OperationMode.CALIBRATION,
+    ) -> CollectionStatus:
+        """Evaluate detections mad ein evaluation mode."""
```

Comment:
mad ein

Reply:
thanks, i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e